apis/kueue/v1alpha1/multikueue_types.go

```diff
@@ -27,6 +27,10 @@ const (
 	// MultiKueueOriginLabel is a label used to track the creator
 	// of multikueue remote objects.
 	MultiKueueOriginLabel = "kueue.x-k8s.io/multikueue-origin"
+
+	// MultiKueueControllerName is the name used by the MultiKueue
+	// admission check controller.
+	MultiKueueControllerName = "kueue.x-k8s.io/multikueue"
```

Comment:
@trasc Does this mean the arbitrary cluster is managed by a single kueue-controller-manager, right?
In other words, the external manager can not deploy Jobs to clusters managed by the `"kueue.x-k8s.io/multikueue"`?

Reply:
More or less yes, and this one aspect we'll work on #2349, the idea for now is to nave the `controllerName` used by a multikueue admission check controller set before starting it and add a `controllerName` field added to the 
`MultiKueueCluster`, with this multiple multikueue admission check controllers using different `controllerName` can work in the same cluster at the same time each on it's own subset of objects (Workloads, AdmissionChecks, MultikueueClusters ).

Reply:
I see this was removed from this PR. That seems appropriate. Resolving for now.